core/src/test/java/io/questdb/test/log/LogAlertSocketTest.java

```diff
@@ -172,7 +172,7 @@ public void testFailOverSingleHost() throws Exception {
             final SOCountDownLatch haltLatch = new SOCountDownLatch(1);
             final CyclicBarrier startBarrier = new CyclicBarrier(2);
             final MockAlertTarget server = new MockAlertTarget(
-                    0,
+                    9863,
```

Comment:
just for my curiosity: is this meant to fix the test flakiness? how?

Reply:
Yes, I ran it on my Mac and with 0 (random) I got a sort of port exhaustion at some point, so decided to pin the port